src/db/schema.rs

```diff
@@ -0,0 +1,24 @@
+table! {
+    proposals (id) {
+        id -> Integer,
+        proposal_category -> Text,
+        proposal_id -> Text,
+        proposal_title -> Text,
+        proposal_summary -> Text,
+        proposal_problem -> Text,
+        proposal_solution -> Text,
+        proposal_funds -> Integer,
```

Comment:
probably want this `BigInt` (~i64) not `Integer`(~i32)

Reply:
Yeah, I need to check this. Initially, I had other types but I got some errors when using them from the rust interface. Thanks!